demos/projects/ESPRESSIF/esp32/components/coreMQTT/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(ROOT_PATH
+    ${CMAKE_CURRENT_LIST_DIR}/../../../../../..
+)
+
+set(AZURE_IOT_MIDDLEWARE_FREERTOS
+    ${ROOT_PATH}/libs/azure-iot-middleware-freertos
+)
+
+set(CORE_MQTT_PATH 
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/libraries/coreMQTT/source
+)
+
+file(GLOB COMPONENT_SOURCES
+    ${CORE_MQTT_PATH}/*.c  
+)
+
+set(COMPONENT_INCLUDE_DIRS
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/.github/config
```

Comment:
this should not be the config pass to coreMqtt

Reply:
Removed.